### ISSUE TITLE
Update quickstart docs with API key creation

### ISSUE THREAD

**Issue by pdeziel** [rotationalio/ensign#756]
This issue relates to the PyEnsign issue opened here: https://github.com/rotationalio/pyensign/issues/72.

The quickstart guide in the docs is missing some info about how to actually create a project, topic, and API key. We would like to add a quick step-by-step guide for creating a project, topic, and API key on the quickstart page (the `getting_started.md`).

**Comment by Mikerniker** [comment]
Hello, yes I'd like to be assigned to this, thank you

**Comment by pdeziel** [comment]
> Hi @pdeziel, I can't seem to run `hugo serve`, I'm getting the following error:
> 
> ```
> Error: error building site: render: failed to render pages: render of "page" failed: "D:\HacktoberPyensign\ensign\docs\themes\godocs-2\layouts\_default\baseof.html:30
> :7": execute of template failed: template: _default/single.html:30:7: executing "_default/single.html" at <partial "essentials/header.html" .>: error calling partial:
>  "D:\HacktoberPyensign\ensign\docs\themes\godocs-2\layouts\partials\essentials\header.html:100:4": execute of template failed: template: partials/essentials/header.ht
> ml:100:4: executing "partials/essentials/header.html" at <partial "components/search-modal" .>: error calling partial: execute of template failed: html/template:parti
> als/components/search-modal.html: ends in a non-text context: {stateJSBqStr delimNone urlPartNone jsCtxRegexp attrNone elementScript <nil>}
> ```
> 
> ChatGPT is telling me that there's an issue with how some HTML or JavaScript is being generated or processed within the template. I'm not very familiar with JavaScript, but I can see that the `id` for the HTML in line 54 and 57 is duplicated:
> 
> ![headerhtml](https://user-images.githubusercontent.com/63586831/272312723-b3a3bd45-a4e7-44f1-b670-afa74cb7f049.PNG)
> 
> I'm not sure if that's the main reason for the error, let me know if you have a recommendation on how I can fix or work around this. Thank you for your help!

@Mikerniker Hmm I haven't seen that error before. Have you made any changes in your fork yet? Or do you think it's just a hugo issue on your end? You might try updating hugo and see if that fixes it.

**Comment by Mikerniker** [comment]
Hi @pdeziel, I haven't made changes to my fork yet. I mainly just installed and tried to follow the set up for Hugo (from the ensign and Hugo docs). I did try creating my own Hugo website following the Hugo Quickstart in another/separate directory to see if that would work--> I was able to successfully run a local Hugo development server with `hugo server -D` with no issues, but can't seem to do the same with the Ensign fork (because of the html/JS error message) I also tried updating Hugo, and I already have the current version v0.119...not sure what else to try... 

As an alternative, I can copy the draft of the Ensign quickstart guide I made with markdown into the Ensign fork and submit a PR if you want to check and give feedback for edits (but I can't currently see how it would look when rendered since I can't seem to run it locally due to the error message I get. Let me know if that also works for you.
